orchagent/aclorch.cpp

```diff
@@ -627,7 +639,19 @@ bool AclRule::remove()
 
     decreaseNextHopRefCount();
 
-    res = removeRanges();
+    return true;
+}
+
+bool AclRule::remove()
+{
+    SWSS_LOG_ENTER();
+
+    if (!removeRule())
+    {
+        return false;
+    }
+
+    auto res = removeRanges();
```

Comment:
@stepanblyschak not sure if it's ok to use `auto` here...

Reply:
This is questionable. As a rule of thumb I would prefer the code with auto, especially considering that return type of the function can be also deduced in newer c++.  I don't see a value in having to explicitelly say it's bool.